src/components/selection-list/basic-list.tsx

```diff
@@ -0,0 +1,112 @@
+import React = require('react');
+import ReactDOM  = require('react-dom');
+import {SBComponent} from 'stylable-react-component';
+import {root} from 'wix-react-tools';
+import {noop} from '../../utils';
+import listStyle from './selection-list.st.css';
+
+export type ItemValue = string;
+
+function closestElementMatching(
+    predicate: (element: HTMLElement) => boolean,
+    startAt: HTMLElement
+): HTMLElement | null {
+    let current: HTMLElement | null = startAt;
+    while (current && !predicate(current)) {
+        current = current.parentElement;
+    }
+    return current;
+}
+
+export function getFocusableItemValues(nodes: React.ReactNode): ItemValue[] {
```

Comment:
You're right, ListNavigation is probably the only one who's ever going to need to know about the list of focusable items. What it shouldn't know about is how each item is structured. It shouldn't be the one to figure out if the item is focusable or not. Currently there's no clean way to separate these responsibilities.

I think we're going to need some kind of ListModel to encapsulate that logic, because DropDown will need to do queries on the list even without creating <SelectionList>. For example for type-ahead. And it would less than ideal to run `renderItem` for each list entry just to determine the item's value :(